src/python/packages/amwg/amwg.py

```diff
@@ -4290,7 +4294,7 @@ def _results(self, newgrid=0):
         #pdb.set_trace()
         results = plot_plan._results(self, newgrid)
         if results is None:
-            print "WARNING, AMWG plot set 12 found nothing to plot"
+            print "WARNING, AMWG plot set 14 found nothing to plot"
```

Comment:
@painter1 I know it's not you but please can you move this one to the logger as well